TrivialSerialisation/Common/BuildFile.xml

```diff
@@ -0,0 +1,4 @@
+<use name="FWCore/Framework"/>
```

Comment:
I think you should declare also a dependency on `DataFormats/Common` due to the use of `edm::Wrapper<T>`.

And maybe also on `FWCore/PluginManager` ?